modules/op-release-notes-1-7.adoc

```diff
@@ -0,0 +1,307 @@
+// https://github.com/tektoncd/triggers/pull/1227
+
+* With this update, the `TriggerGroups` field is added to the `EventListener` specification. Using this field, you can specify a set of interceptors for execution, before a group of triggers are selected and executed. To enable this feature, set the `enable-api-fields` flag in the `feature-flags-triggers` config map to `alpha`.
+// https://github.com/tektoncd/triggers/pull/1232
+
+* With this update, Triggers support custom runs defined by a `TriggerTemplate` template.
+// https://github.com/tektoncd/triggers/pull/1283/files
+
+* With this update, Triggers support emitting Kubernetes events from an `EventListener` pod.
+// https://github.com/tektoncd/triggers/pull/1222
+
+* With this update, count metrics are available for the following objects: `ClusterInteceptor`, `EventListener`, `TriggerTemplate`, `ClusterTriggerBinding`, and `TriggerBinding`.
+// https://github.com/tektoncd/triggers/pull/1305
+
+* With his update, the `ServicePort` specification to Kubernetes resource is added. You can use it to modify on which port the event listener service is exposed. The default port is `8080`.
+// https://github.com/tektoncd/triggers/pull/1272
+
+* With this update, you can use the `targetURI` field in the `EventListener` specification to send cloud events during trigger processing. To enable this feature, set the `enable-api-fields` flag in the `feature-flags-triggers` config map to `alpha`.
+// https://github.com/tektoncd/triggers/pull/1259
+
+* With this update, the `tekton-triggers-eventlistener-roles` object now has a `patch` verb, in addition to the `create` verb that already exists.
+// https://github.com/tektoncd/triggers/pull/1291
+
+* With this update, the `securityContext.runAsUser` parameter is removed from event listener deployment.
+// https://github.com/tektoncd/triggers/pull/1213
+
+[id="cli-new-features-1-7_{context}"]
+=== CLI
+
+* With this update, the `tkn [pipeline | pipelinerun] export` command will export a pipeline or pipeline run as a YAML file. For example:
+** Export a pipeline named `test_pipeline` in the `openshift-pipelines` namespace:
++
+[source,terminal]
+----
+$ tkn pipeline export test_pipeline -n openshift-pipelines
+----
+** Export a pipeline run named `test_pipeline_run` in the `openshift-pipelines` namespace:
++
+[source,terminal]
+----
+$ tkn pipelinerun export test_pipeline_run -n openshift-pipelines
+----
+// https://github.com/tektoncd/cli/pull/1398 and https://github.com/tektoncd/cli/pull/1500
+
+* With this update, the `--grace` option is added to the `tkn pipelinerun cancel`. Use the `--grace` option to gracefully terminate a pipeline run, instead of forced termination. To enable this feature, set the `enable-api-fields` flag in the `feature-flags` config map to `alpha`.
```

Comment:
```suggestion
* With this update, the `--grace` option is added to the `tkn pipelinerun cancel`. Use the `--grace` option to terminate a pipeline run gracefully instead of forcing the termination. To enable this feature, set the `enable-api-fields` flag in the `feature-flags` config map to `alpha`.
```